### PR TITLE
Entry form styling

### PR DESCRIPTION
**Pull request by brew**
This PR implements the "Implement front-end design for entry form" portion of #753.

* Reorganise template directories and styling
  - Move old views from `views` --> `views_old` directory
  - New views in `views` directory
  - Adds supporting assets and bower_components to new `static` directory (old `public` directory is now legacy).

* Entry form changes
  - Pre-render QuestionForm React component in controller action
  - Yes/No question type styling
  - Supporting fields; comments and anonymous attribution choice.

Entry form fields are all of the yes/no choice type, regardless of actual type. This will be addressed in a subsequent PR.

* Ancillary pages use new design; About, FAQ etc.

### CODE REVIEW

**Comment by pwalsh** [comment]
Brook just a note, on new projects over the last year we have completely dropped bower and use npm for front end assets too. I think it is preferable as it makes it easier to use the same build tools and so on. Generally, we compile vendor dependencies into a single vendor.min.js and our own code into app.min.js

**Comment by brew** [comment]
Good to know. Can you point me to an example?

**Comment by pwalsh** [comment]
@brew sure:

- single app.min: https://github.com/frictionlessdata/data-quality-dashboard/blob/master/webpack.config.production.js#L10
- more complex example, also building a single dist file (I think): https://github.com/opentrials/opentrials/blob/master/webpack.config.js#L7
- another single dist: https://github.com/openspending/babbage.ui/blob/master/webpack.config.production.js#L8
- distinct vendor + app dist files: https://github.com/frictionlessdata/datapackagist/blob/master/gulpfile.js#L69

Seeing as we have moved to webpack (last example is gulp), and, it seems we are all building a single dist with webpack (no distinct vendor + app files), that seems the way to go.

**Comment by brew** [comment]
Great, thanks!

**Comment by brew** [comment]
An example of the new entry form can be seen here: http://global-test.next-census.okfn.org/submit-react/?place=al&dataset=timetables

Note, the main body of questions from the CMS, but the form doesn't currently submit.

**Comment by pwalsh** [comment]
Looks great on mobile!

File: census/views/base.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% set godi = "false" %}
+{% set godi = site.settings.is_godi or false %}
```
Comment (pwalsh) [inline census/views/base.html]: @brew `godi` is too specific ("global"), I'd say the flag can be `is_index` ("Survey" or "Index").

**Pull request merged**